src/test/java/edu/wisc/my/restproxy/service/RestProxyServiceImplTest.java

```diff
@@ -60,12 +63,33 @@ public void proxyRequest_control() {
   }
   
   /**
+   * Test simulates a proxy request which fails with a http 400 error.  
+   * An error like this could be encountered if you were to post invalid data to a form.
+   * Test verifies the HttpStatus AND the body (which likely contins validation error message) are passed back to the client.
+   */
+  @Test
```

Comment:
I call BS on these unit tests... I replaced the whole `proxyRequest` method with

```
return proxyDao.proxyRequest(null);
```

and the test passes! :P  Some other tests in here make underhanded assertion via the mock matchers themselves, which is not better, in my opinion.

I wouldn't want to hold up the PR, but I'm making a note to add integration tests on the standalone service, or if possible, to refactor such that this much mocking is not needed.